### ISSUE TITLE
npm WARN

### ISSUE THREAD

**Issue by michaelachrisco** [dphaener/reactable-forms#2]
Looking over the repo and wanted to give it a try. Thought I would let you know about this WARN:

``` bash
michaelchrisco@:~/development/reactable-forms\ (master)$ npm install
npm WARN peerDependencies The peer dependency react included from reactable-forms will no
npm WARN peerDependencies longer be automatically installed to fulfill the peerDependency 
npm WARN peerDependencies in npm 3+. Your application will need to depend on it explicitly.
```

**Comment by dphaener** [comment]
@michaelachrisco Yeah I know. Laziness. :) Thanks for the reminder. :+1:

**Comment by michaelachrisco** [comment]
@dphaener I thought it was my local machine. Carry on :8ball:

**Comment by dphaener** [comment]
Re-opening. Referenced by the wrong PR.

**Comment by dphaener** [comment]
Fixed by b69c1c3026915a172e42ee911efe95b32cb9f3dd
